src/main.ts

```diff
@@ -23,13 +23,19 @@ async function run() {
 
         core.addPath(path.join(process.cwd(), 'bin'));
 
-        // For some reason, this makes the action step to never finish on
-        // Windows
-        // console.log("Built version:");
-        // await exec.exec(`alr -n version`);
+        // For some reason, this makes the action step to never finish on Windows
+        if (process.platform != "win32") {
+            console.log("Built version:");
+            await exec.exec(`alr -n version`);
+        }
         console.log("SUCCESS");
     } catch (error) {
-        core.setFailed(error.message);
+        if (error instanceof Error) {
+            core.setFailed(error.message);
+        }
+        else {
+            core.setFailed("Unknown error situation");
+        }
```

Comment:
This change will likely be eventually needed for the stable branch, to keep in mind